Userland/Applications/Spreadsheet/SpreadsheetView.cpp

```diff
@@ -173,16 +174,32 @@ void InfinitelyScrollableTableView::mousemove_event(GUI::MouseEvent& event)
 
 void InfinitelyScrollableTableView::mousedown_event(GUI::MouseEvent& event)
 {
-    if (m_is_hovering_cut_zone)
+    // Note: Override the mouse event so that the the cell that is 'clicked' is not
```

Comment:
Since this seems to be (more properly?) implementing the functionality that the previous commit removed as useless variables, just squash the two together.

Reply:
Also, the commit message needs some love.
e.g. "Select the correct cell on click"